addons/test_mail/tests/test_performance.py

```diff
@@ -1434,7 +1434,7 @@ def test_message_to_store_multi(self):
         """
         messages_all = self.messages_all.with_env(self.env)
 
-        with self.assertQueryCount(employee=23):  # tm 22
+        with self.assertQueryCount(employee=24):  # tm 23
```

Comment:
Note: this query count is incremented due to the addition of the `thumbnail` field on `ir.attachment`

Reply:
This is the query that is added:

```sql
SELECT "ir_attachment"."id" FROM "ir_attachment" WHERE ("ir_attachment"."res_field" IN ('thumbnail') AND "ir_attachment"."res_id" IN (1709, 1710, 1712, 1711, 1714, 1713, 1716, 1715) AND "ir_attachment"."res_model" IN ('ir.attachment')) ORDER BY "ir_attachment"."id" DESC  
```

It's because of the has_thumbnail compute that is computed because we access the field inside the `_to_store_defaults` method.